src/main/java/com/algorand/algosdk/transaction/Transaction.java

```diff
@@ -324,18 +324,33 @@ public Transaction() {}
         AssetConfig("acfg"),
         AssetFreeze("afrz");
 
+        private static Map<String, Type> namesMap = new HashMap<String, Type>(5);
+
+        static {
+            namesMap.put(Default.value, Default);
```

Comment:
A check for duplicates is not necessary since the initialization of the private map only occurs here.  Case sensitivity is preserved per the previous comment.